### PR TITLE
Search result metadata display logic

### PR DESCRIPTION
**Pull request by rwd**
* Display eng, en or def values if present; else display raw value
* Remove duplicates
* Remove URIs if other values are present
* Output multiple values as list

### CODE REVIEW

File: pages/search/index.vue [hunk 1]
```diff
@@ -103,11 +114,51 @@
 <script>
   import LoadingSpinner from '~/components/LoadingSpinner.vue';
   import axios from 'axios';
+  // TODO: cherry-pick Lodash methods? or load from CDN?
+  import _ from 'lodash';
```
Comment (mirjamverloop) [inline pages/search/index.vue]: Maybe it's worth to investigate if we need lodash at all. [This](https://github.com/you-dont-need/You-Dont-Need-Lodash-Underscore) is a handy overview.

**Comment by rwd** [comment]
@lbiedinger re:
> Down the line something that's quite apparent is missing labels.

Do you mean field labels like "creator"? If so, we may very well not have them down the line on a search results page.

File: pages/search/index.vue (hunk 1 above)
Comment (rwd) [reply pages/search/index.vue]: Where native equivalents exist, they have been switched to. Where not, Lodash's have been cherry-picked.

**Comment by lbiedinger** [comment]
> Do you mean field labels like "creator"? If so, we may very well not have them down the line on a search results page.

Yes, creator & providing institution, or perhaps "by" and "from". But yes, until it's clear if and how these should be displayed let's not add anything.

**Pull request merged**